src/cli/argument-parser.ts

```diff
@@ -273,6 +275,11 @@ export default class CLIArgumentParser {
             this.opts.concurrency = parseInt(this.opts.concurrency as string, 10);
     }
 
+    private async _parseQuarantineOptions (): Promise<void> {
+        if (this.opts.quarantineMode && this.opts.quarantineMode.length > 0)
```

Comment:
As I understand, here the `this.opts.quarantineMode` is string. If so, we do not need to check the `length` property.